lesson_5/homeTask/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>lesson_5 - HomeTask</title>
+  </head>
+  <body>
+    <div><h1>Mazniak Mykhailo</h1></div>
+    <div>
+      <img
+        src="C:/Robot_Dreams/lesson_4/image/Foto.jpg"
+        alt="Foto for CV"
+        width="250"
+        height="350"
+      />
+      <h4>
+        Student of course Front-End developer
+        <a
+          href=" https://robotdreams.cc/uk/course/osnovy-frontend-razrabotki-html-css-javascript"
+        >
+          robot_dreams
+        </a>
+      </h4>
+
+      <div>
+        <h4>
+          <i><u>Work Experince</u></i>
+        </h4>
+        <table border="1">
+          <thead>
+            <tr>
+              <th>Period</th>
+              <th>Position</th>
+            </tr>
+          </thead>
+          <tr>
+            <th align="left">04.2023 ...</th>
+            <td>
+              Operator CNC at
+              <a href="http://www.stemmann.com.pl/index.pl.php"
+                >Stemmann-Polska</a
```

Comment:
bad formatting, should like these:
```
<a href="http://www.stemmann.com.pl/index.pl.php">
  Stemmann-Polska
</a>
```

Reply:
I know, but my Visual Studio (Prettier) does it past save